lib/src/hitmap.dart

```diff
@@ -157,7 +180,29 @@ Future<Map<String, HitMap>> createHitmap(
 }
 
 /// Merges [newMap] into [result].
-void mergeHitmaps(Map<String, HitMap> newMap, Map<String, HitMap> result) {
+///
+/// DEPRECATED: Migrate to mergeHitmapsV2.
+void mergeHitmaps(
+    Map<String, Map<int, int>> newMap, Map<String, Map<int, int>> result) {
+  newMap.forEach((String file, Map<int, int> v) {
```

Comment:
```suggestion
  newMap.forEach((file, v) {
```

https://dart.dev/guides/language/effective-dart/design#dont-annotate-inferred-parameter-types-on-function-expressions